src/ImmersedBoundaries/grid_fitted_immersed_boundaries.jl

```diff
@@ -44,8 +44,8 @@ end
 GridFittedBottom(bottom_height) = GridFittedBottom(bottom_height, CenterImmersedCondition())
 
 function Base.summary(ib::GridFittedBottom)
-    hmax = maximum(ib.bottom_height)
-    hmin = minimum(ib.bottom_height)
+    hmax = maximum(parent(ib.bottom_height))
+    hmin = minimum(parent(ib.bottom_height))
```

Comment:
What's the issue?

Reply:
scalar indexing when `ib.bottom_height` is an `OffsetArray{CuArray}`

Reply:
Ah, I forgot that it wasn't a field.